code/style.css

```diff
@@ -3,17 +3,24 @@
 }
 
 body {
+  background-image: url("https://images.unsplash.com/photo-1485809722779-439bb523521d?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1500&q=80");
```

Comment:
It would be much nicer if your background covers a whole screen, not only 1/4 🧑‍🎨